ci/scripts/common.sh

```
@@ -146,7 +158,17 @@ function create_env() {
 
     rapids-logger "Creating Environment with extras: ${@}"
 
+    set +e
     UV_SYNC_STDERROUT=$(uv sync --active ${extras[@]} 2>&1)
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)